src/Microsoft.Health.Fhir.SqlServer/Features/Search/SortUtils.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Core.Features;
+using Microsoft.Health.Fhir.Core.Features.Search;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search
+{
+    public static class SortUtils
+    {
+        private static readonly IList<string> SupportedParameters = new List<string>
+            {
+                KnownQueryParameterNames.LastUpdated,
+            };
+
+        public static SortOrder GetSortOrderForSupportedParam(SearchOptions searchOptions)
```

Comment:
I think we're good on functionality in this PR, just a few more thoughts on this class.
 - Add some documentation for what this function is doing
 - Probably a good candidate for an extension method, also rename so its more clear what its doing `public static SortOrder GetFirstSortOrderForSupportedParam(this SearchOptions searchOptions)`

Reply:
Good point. Fixed